src/test/java/com/rocketden/main/UserTests.java

```diff
@@ -126,4 +130,33 @@ public void createNewUserNicknameContainsSpaces() throws Exception {
 
         assertEquals(ERROR.getResponse(), actual);
     }
+
+    @Test
+    public void deleteExistingUser() throws Exception {
+        CreateUserRequest createRequest = new CreateUserRequest();
+        createRequest.setNickname("rocket");
+        
+        DeleteUserRequest request = new DeleteUserRequest();
+        request.setNickname("rocket");
+
+        this.mockMvc.perform(post(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)));
+
+        this.mockMvc.perform(delete(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)))
+                .andExpect(status().isAccepted());
+    }
+
+    @Test
+    public void deleteNonExistingtUser() throws Exception {
+        DeleteUserRequest request = new DeleteUserRequest();
+        request.setNickname("rocket");
+
+        this.mockMvc.perform(delete(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)))
+                .andExpect(status().isNotFound());
```

Comment:
Same as my other comment, I'd suggest adding a check that the returned error message is correct (see the other tests)